planner/core/src/main/java/eu/seaclouds/platform/planner/core/Planner.java

```diff
@@ -296,32 +309,31 @@ public SLAInfo() {}
 
     }
 
-    class OfferJson {
+    public static class Offering {
```

Comment:
Not really. It is inside Planner since it is used by this class. Scattering inner classes in actual different file does not seem to be a god practice, although Java encourage to do that.